notifications/src/main/java/dk/shape/games/notifications/presentation/SubjectNotificationsFragment.kt

```diff
@@ -141,4 +155,7 @@ class SubjectNotificationsFragment : BottomSheetDialogFragment() {
             .inflate(layoutInflater)
             .apply { viewModel = notificationsSheetViewModel }.root
     }
+
+    private fun BottomSheetDialog.toBottomSheetView(): ViewGroup? =
+        findViewById<View>(com.google.android.material.R.id.design_bottom_sheet) as? ViewGroup
```

Comment:
You can also just create a require like this:

```suggestion
        private fun BottomSheetDialogFragment?.requireBottomSheetView(): ViewGroup? {
        return (this?.dialog as? BottomSheetDialog?)?.findViewById<View>(R.id.design_bottom_sheet) as? ViewGroup?
    }
```

Reply:
Then you can use it on the listener like this:

```kotlin
private val viewSwitcherViewModel: SubjectNotificationSwitcherViewModel =
        SubjectNotificationSwitcherViewModel {
            requireBottomSheetView()?.let {
                TransitionManager.beginDelayedTransition(
                    it,
                    AutoTransition().setInterpolator(FastOutSlowInInterpolator())
                )
            }
        }
```